src/app/admin-sites-page/admin-sites-page.component.ts

```diff
@@ -19,13 +19,6 @@ export class AdminSitesPageComponent implements OnInit {
     this.showModal = false;
     this.selectedSite = {};
     this.siteService.getSitesByRole('admin').pipe(first()).subscribe((sites) => {
-      sites.forEach(site => {
```

Comment:
@darrengsmithson why is this removed ?

Reply:
we no longer need to call site service for each site to get the parent info